src/plugins/data/public/ui/query_string_input/query_string_input.tsx

```diff
@@ -56,7 +56,10 @@ export interface QueryStringInputProps {
   size?: SuggestionsListSize;
   className?: string;
   isInvalid?: boolean;
+  isClearable?: boolean;
   iconType?: string;
+  nonKqlMode?: 'lucene' | 'text';
```

Comment:
Could you please add a short tsDoc description on top of this prop? 🙏 
took me some time to understand what it does - wasn't clear from just the name

Reply:
done in [`e8ec7a1` (#91814)](https://github.com/elastic/kibana/pull/91814/commits/e8ec7a11a7034af5bd939913088fe2aacc874e7c)